torch_xla/experimental/tpu.py

```diff
@@ -81,10 +80,28 @@ def task_id() -> Optional[int]:
   return xu.getenv_as(xenv.CLOUD_TPU_TASK_ID, int)
 
 
+def using_env_vars() -> bool:
```

Comment:
nit: prefix this function with an underscore because it shouldn't be used outside of this module

Reply:
Done